Deploy to GitHub Pages

norbiboy1 opened the pull request:
In this pull request I have achieved the following:

1.Deployed the page using GitHub Pages
2.Updated the README file to include the page link

Commit:
Add some animations

principlebrothers committed:
Add animation to all button

Commit:
Animation work

Commit:
Linter error configuration to access deployment

dasileker reviewed (changes requested):
### **Hi @norbiboy1,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**

- everything is awesome✅.
- Linters are Good✅.
- The page is deployed ✅.


## Required Changes ♻️
_Check the comments under the review._

- [ ] nice job on deploying your Portfolio, however, your readme file still has to work on it, and also you need to add your deployed link on the readme file.

1. getting started should have the direction on how to clone and how open the app.
2. add deployed link on the readme file. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

norbiboy1 committed:
Link to the GitHub Page

You can see my project live.

norbiboy1 committed:
Merge branch 'main' into deploy

Temmarie reviewed (changes requested):
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @norbiboy1    ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You have successfully deployed the app✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Recommended** ♻️
- [ ] As requested by the previous reviewer, kindly add the proper steps required for a user to clone your project.

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._**
__Kindly make the necessary changes and submit another code review request.__

norbiboy1 committed:
Local copy and desktop app starting with a description.

You can see the README.md and can copy on local operation system to watch the code and see how I configured the webpage.

norbiboy1 committed:
Deploy to GitHub Pages

1. Add a link to Live Demo section.
2. Getting Started section you must read to follow the setup to clone it in your own local environment.

Kossi-stack reviewed (approved):
## Status: Approved ✔️
**Hi @norbiboy1,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉

![](https://user-images.githubusercontent.com/74506933/148243736-d9f63735-4cb1-411f-be28-34972618852d.gif)
You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

norbiboy1 committed:
Update README.md
